src/swarm/neo/connection/ConnectionBase.d

```diff
@@ -785,6 +785,19 @@ abstract class ConnectionBase: ISelectClient
         return *client_address.set(this.socket.in_addr);
     }
 
+    /***************************************************************************
+
+        Resets all cumulative stats counters for this connection.
+
+    ***************************************************************************/
+
+    public void resetStatsCounters ( )
+    {
+        this.send_loop.queue = this.send_loop.queue.init;
```

Comment:
This looks wrong, do you mean `this.send_loop.queue.stats`?

Reply:
Oops, yes.

Reply:
Updated.